src/patterns/question-pages/default.njk

```diff
@@ -15,8 +15,14 @@ layout: layout-example.njk
   <div class="govuk-o-main-wrapper">
     <div class="govuk-o-grid">
       <div class="govuk-o-grid__item govuk-o-grid__item--two-thirds">
-
-        <form class="form" action="/url/of/next/page" method="post">
+      <!--
+        This example omits the 'action' and uses the 'get' method so that
+        submitting the form just reloads the example. In most cases, your form tag
+        will probably need to look something like this:
+
+        <form action="/url/of/next/page" method="post">
+      -->
+        <form method="get">
```

Comment:
As mentioned in comments, I think this change would cause problems for prototyping, where POST is safer and more correct.